src/components/Video/VideoProperties.vue

```diff
@@ -21,7 +21,7 @@
               </p>
             </div>
           </div>
-          <p v-else-if="recording[prop.key]" >
+          <p v-else-if="recording[prop.key] != null" >
```

Comment:
this happens to cover `undefined` too much is great

https://stackoverflow.com/a/5515385